go/private/rules/wrappers.bzl

```diff
@@ -25,6 +25,8 @@ _CGO_ATTRS = {
     "srcs": None,
     "cdeps": [],
     "copts": [],
+    "cxxopts": [],
+    "cppopts": [],
```

Comment:
Please update attribute tables in go/core.rst to include the new attributes.

This may break people that have C++ code that currently depends on options in `copts`. Hopefully this won't affect many people. I'll file an issue on Gazelle to separate these.